### PR TITLE
fix: update read the docs for balsamic v12.0.0

### PR DESCRIPTION
**Pull request by mathiasbio**
### This PR:

Updates read the docs for release 12.0.0 of balsamic.

## Changes in the latest release of balsamic for reference:

**Added:**
- PIP specific missing tools to config https://github.com/Clinical-Genomics/BALSAMIC/pull/1096
- Filtering script to remove normal variants from TIDDIT feat: add TIDDIT tumor normal filtering  #1120
- Store TMB files in HK https://github.com/Clinical-Genomics/BALSAMIC/pull/1144

**Changed:**
- Fixed all conda container dependencies https://github.com/Clinical-Genomics/BALSAMIC/pull/1096
- Changed --max_sv_size in VEP params to the size of chr1 for hg19 https://github.com/Clinical-Genomics/BALSAMIC/pull/1124
- Increased time-limit for sambamba_exon_depth and picard_markduplicates to 6 hours https://github.com/Clinical-Genomics/BALSAMIC/pull/1143
- Update cosmicdb to v97 perf: COSMIC database to v97  #1147

**Fixed:**
- Update cryptography version (39.0.1) due to security alert https://github.com/Clinical-- Genomics/BALSAMIC/pull/1087
- Pytest file saved in balsamic directory fix: no more pytest generated files in my directory  #1093
- Fix varcall_py3 container bcftools dependency error https://github.com/Clinical-Genomics/BALSAMIC/pull/1097

## Changes in the read the docs made here:

**Added:**
- Descriptions of which filters were added in WGS T / N for TIDDIT normal filtering
- Update about maximum sv-size for VEP annotation

**Removed:**
- WGS from list of sequencing types where VarDict is used (just a correction)

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: docs/balsamic_filters.rst
Comment (ivadym) [inline docs/balsamic_filters.rst:1]: Nice! 

I have also spotted some erros: Dnascope is TGA and WGS, TNscope UMI is only TGA and NOT germline 🤦 

In atlas is correct: https://atlas.scilifelab.se/production/data_analysis/workflows/balsamic/#data-delivery

File: CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ Changed:
 * Changed --max_sv_size in VEP params to the size of chr1 for hg19 https://github.com/Clinical-Genomics/BALSAMIC/pull/1124
 * Increased time-limit for sambamba_exon_depth and picard_markduplicates to 6 hours https://github.com/Clinical-Genomics/BALSAMIC/pull/1143
 * Update cosmicdb to v97 https://github.com/Clinical-Genomics/BALSAMIC/pull/1147
+* Updated read the docs with the changes relevant to mention
```
Comment (ivadym) [inline CHANGELOG.rst:16]:
```suggestion
* Updated read the docs with the changes relevant to mention https://github.com/Clinical-Genomics/BALSAMIC/pull/1153
```

**Review by ivadym — APPROVED** [review]
Fantastic @mathiasbio 🥳

File: docs/balsamic_filters.rst
Comment (mathiasbio) [reply docs/balsamic_filters.rst:1]: Nice catch, I'll make the changes!

File: CHANGELOG.rst (hunk 1 above)
Comment (mathiasbio) [reply CHANGELOG.rst]: Right! Added!

**Pull request merged**